Tests/Unit/NetworkServer/MessageProcessorSingleGatewayTest.cs

```diff
@@ -147,7 +140,7 @@ public async Task Unknown_Region_Should_Return_Null()
                 deviceRegistry,
                 FrameCounterUpdateStrategyProvider);
 
-            using var request = CreateWaitableRequest(rxpk, useRealTimer: true);
+            using var request = CreateWaitableRequest(TestUtils.GenerateTestRadioMetadata(frequency: 0), payload, useRealTimer: true);
```

Comment:
2\. Belongs with suggestion 1.

```suggestion
            using var request = CreateWaitableRequest(TestUtils.GenerateTestRadioMetadata(frequency: new Hertz(0)), payload, useRealTimer: true);
```